dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -678,7 +678,7 @@
 		};
 
 		dma2: dma@40020400 {
-			compatible = "st,stm32-dma";
+			compatible = "st,stm32-dma-v2";
 			#dma-cells = <4>;
 			reg = <0x40020400 0x400>;
 			interrupts = <11 0>, <12 0>, <13 0>, <14 0>, <15 0>, <16 0>,
```

Comment:
not really related to this, but the `interrupts` property should be `interrupts = <56 0>, <57 0>, <58 0>, <59 0>, <60 0>, <68 0>, <69 0>, <70 0>;`

Reply:
Thx for catching that. I've fixed this as well.